File: fizzbuzz.php
<?php

/*
FizzBuzz
Neste problema, você deverá exibir uma lista de 1 a 100, um em cada linha, com as seguintes exceções:

Números divisíveis por 3 deve aparecer como 'Fizz' ao invés do número;
Números divisíveis por 5 devem aparecer como 'Buzz' ao invés do número;
Números divisíveis por 3 e 5 devem aparecer como 'FizzBuzz' ao invés do número'.
*/

for ($cont=1; $cont<101; $cont++){
    echo "<br>";
    if(($cont % 3 == 0) || ($cont % 5 == 0)){
    if($cont % 3 == 0){
        echo "Fizz";
    }
    if($cont % 5 == 0){
        echo "Buzz";
    }
    }else{
        echo $cont;
    }
}


?>

File: FizzBuzz.java
package segundodesafio;

import java.util.Scanner;

public class FizzBuzz {
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		
		for(int i = 0; i <= 100; i++) {
			
			if(i%3 == 0) {
				System.out.println("Fizz"+i);
			}
			if(i%5 == 0) {
				System.out.println("Buzz"+i);			
			}
			if((i%3 == 0) && (i%5 ==0)) {
				System.out.println("FizzBuzz"+i);
		
		
	 		}	
	    }
    }
}